packages/jest-cli/src/TestRunner.js

```diff
@@ -439,14 +443,13 @@ class ReporterDispatcher {
     );
   }
 
-  onTestResult(config, testResult, results) {
+  onTestResult(test, testResult, results) {
```

Comment:
Same, Flow isn't asking for it. Is this about readability? If Flow isn't asking it means the type isn't ambiguous to flow.